flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/calls/BridgingFunctionGenUtil.scala

```diff
@@ -308,7 +315,12 @@ object BridgingFunctionGenUtil {
     operands
       .zip(argumentDataTypes)
       .map { case (operand, dataType) =>
-        operand.copy(resultTerm = genToExternalConverterAll(ctx, dataType, operand))
+        operand match {
+          case external: ExternalGeneratedExpression
+            if !isInternal(dataType) && external.getDataType == dataType
+          => operand.copy(resultTerm = external.getExternalTerm, code = external.getExternalCode)
```

Comment:
checkstyle: Indentation should be increased

Reply:
@JingsongLi  Master code changes a lot, I try to resolve conflicts, Please help to review again.